intermediate_notebooks/E2E/taxi/NYCTaxi-E2E.ipynb

```diff
@@ -106,113 +153,208 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [],
    "source": [
     "# helper function which takes a DataFrame partition\n",
-    "def clean(df_part, remap, must_haves):    \n",
+    "from cudf.dataframe import DataFrame\n",
```

Comment:
We want to avoid doing this to prevent any user confusion about what is a cuDF DataFrame and what is a Pandas DataFrame.

Please stick with just `import cudf` for now.